shipper.py

```diff
@@ -126,13 +125,13 @@ def shipment_new():
                     'user': user_entitled,
                     'status': 'new',
                     'action': request.forms.get('action'),
-                    'md': new_md
+                    'md': new_md,
+                    'file_id': request.forms.get('file_id')
```

Comment:
I'm a bit behind concerning the api docu.
`file_id` is only needed, when deleting a specific file from a deposition. if there is only one file in deposition, `file_id` can be `None`.
`action` is the parameter for CRUD actions in repository file management. This is required and probabliy the reason for the above stated "authentication" error. (providing too few parameters made it think, we miss the cookie that is the last parameter.